environments/icds-new/inventory.ini

```diff
@@ -66,14 +66,18 @@ pgucrstandby1
 pgbouncer_max_connections=3500
 pgbouncer_default_pool=775
 postgresql_max_connections=800
-postgresql_work_mem=32MB
-postgresql_shared_buffers=8GB
-postgresql_effective_cache_size=24GB
+postgresql_work_mem=3495kB
```

Comment:
IIRC I set this to 32 MB based on using this https://pgtune.leopard.in.ua/#/ (looks like you may have used this?) and that this was a "data warehouse". The site looks like it has changed quite a bit and we've also upgraded to 9.6 since then. the numbers are quite different just changing 9.4 to 9.6 and 9.6 generally suggests less memory needed. 

So based on that 3.5 MB looks good and safe to me

Reply:
Done https://github.com/dimagi/commcare-cloud/pull/1911/files#diff-0688e9dafa291b12fa97b468ab69dfa1R69